src/db_api_util.ts

```diff
@@ -515,52 +515,37 @@ export async function getSlackThreadsForVoter(
   }
 }
 
-export async function archiveMessagesForDemoVoter(
-  userId: string,
-  twilioPhoneNumber: string
-): Promise<void> {
-  logger.info(`ENTERING DBAPIUTIL.getSlackThreadsForVoter`);
+export async function archiveDemoVoter(userId: string): Promise<void> {
+  logger.info(`ENTERING DBAPIUTIL.archiveDemoVoter`);
```

Comment:
Can you add twilioPhoneNumber back? A unique voter is identified by their userId + twilioPhoneNumber throughout the code, so we should keep that consistent.

Reply:
updated!

Reply:
Are you sure that twilio_phone_number is not always set for voter_status_updates? I'm not seeing a place in the code where it's not set:

https://github.com/tomerovadia/VoterHelpLine/blob/f59d6d5cf8aa8799c44b8398dd91217251c0df70/src/router.ts#L98
https://github.com/tomerovadia/VoterHelpLine/blob/f59d6d5cf8aa8799c44b8398dd91217251c0df70/src/router.ts#L955
https://github.com/tomerovadia/VoterHelpLine/blob/f59d6d5cf8aa8799c44b8398dd91217251c0df70/src/slack_interaction_handler.ts#L162

and the production database shows no rows with it missing

Reply:
```
helpline=> select * from voter_status_updates ;
          created_at           |             user_id              | user_phone_number | twilio_phone_number | is_demo | voter_status | originating_slack_user_name | originating_slack_user_id | slack_channel_name | slack_channel_id | slack_parent_message_ts | action_ts | archived 
-------------------------------+----------------------------------+-------------------+---------------------+---------+--------------+-----------------------------+---------------------------+--------------------+------------------+-------------------------+-----------+----------
 2020-10-25 18:25:55.705072-05 | 0f28ee384ea7e2e1cf4dbfe3bc9f1fb2 | +19512883411      |                     | t       | UNKNOWN      |                             |                           |                    |                  |                         |           | t
 2020-10-25 18:26:24.972818-05 | 0f28ee384ea7e2e1cf4dbfe3bc9f1fb2 | +19512883411      |                     | t       | UNKNOWN      |                             |                           |                    |                  |                         |           | 
```

Reply:
Is that in production? I'm not seeing any null

![Screen Shot 2020-10-25 at 8 53 14 PM](https://user-images.githubusercontent.com/15622817/97123867-27cd3500-1704-11eb-8941-da93fc4fc0a2.png)

I'm also not seeing rows that match that data in production

Reply:
I'm terrified that we almost deployed that bug to production. Can you please double check to make sure everything else looks ready?

Reply:
Checked again and it looks good to me. *Note that this was easy to fix since the field is redundant wrt is_demo)